-------------------------------------------------------------------------------
Test set: dao.Sql2oUserDaoTest
-------------------------------------------------------------------------------
Tests run: 9, Failures: 0, Errors: 9, Skipped: 0, Time elapsed: 0.004 sec <<< FAILURE! - in dao.Sql2oUserDaoTest
addingUserSetsUserId(dao.Sql2oUserDaoTest)  Time elapsed: 0.003 sec  <<< ERROR!
org.sql2o.Sql2oException: Could not acquire a connection from DataSource - No suitable driver found for jdbc:
	at dao.Sql2oUserDaoTest.setUp(Sql2oUserDaoTest.java:29)
Caused by: java.sql.SQLException: No suitable driver found for jdbc:
	at dao.Sql2oUserDaoTest.setUp(Sql2oUserDaoTest.java:29)

addingUserSetsUserId(dao.Sql2oUserDaoTest)  Time elapsed: 0.003 sec  <<< ERROR!
java.lang.NullPointerException
	at dao.Sql2oUserDaoTest.tearDown(Sql2oUserDaoTest.java:35)

addedUserIsReturnedCorrectly(dao.Sql2oUserDaoTest)  Time elapsed: 0 sec  <<< ERROR!
org.sql2o.Sql2oException: Could not acquire a connection from DataSource - No suitable driver found for jdbc:
	at dao.Sql2oUserDaoTest.setUp(Sql2oUserDaoTest.java:29)
Caused by: java.sql.SQLException: No suitable driver found for jdbc:
	at dao.Sql2oUserDaoTest.setUp(Sql2oUserDaoTest.java:29)

addedUserIsReturnedCorrectly(dao.Sql2oUserDaoTest)  Time elapsed: 0 sec  <<< ERROR!
java.lang.NullPointerException
	at dao.Sql2oUserDaoTest.tearDown(Sql2oUserDaoTest.java:35)

getReviewsByUser(dao.Sql2oUserDaoTest)  Time elapsed: 0 sec  <<< ERROR!
org.sql2o.Sql2oException: Could not acquire a connection from DataSource - No suitable driver found for jdbc:
	at dao.Sql2oUserDaoTest.setUp(Sql2oUserDaoTest.java:29)
Caused by: java.sql.SQLException: No suitable driver found for jdbc:
	at dao.Sql2oUserDaoTest.setUp(Sql2oUserDaoTest.java:29)

getReviewsByUser(dao.Sql2oUserDaoTest)  Time elapsed: 0 sec  <<< ERROR!
java.lang.NullPointerException
	at dao.Sql2oUserDaoTest.tearDown(Sql2oUserDaoTest.java:35)

allInstancesAreReturned(dao.Sql2oUserDaoTest)  Time elapsed: 0 sec  <<< ERROR!
org.sql2o.Sql2oException: Could not acquire a connection from DataSource - No suitable driver found for jdbc:
	at dao.Sql2oUserDaoTest.setUp(Sql2oUserDaoTest.java:29)
Caused by: java.sql.SQLException: No suitable driver found for jdbc:
	at dao.Sql2oUserDaoTest.setUp(Sql2oUserDaoTest.java:29)

allInstancesAreReturned(dao.Sql2oUserDaoTest)  Time elapsed: 0.001 sec  <<< ERROR!
java.lang.NullPointerException
	at dao.Sql2oUserDaoTest.tearDown(Sql2oUserDaoTest.java:35)

dao.Sql2oUserDaoTest  Time elapsed: 0.001 sec  <<< ERROR!
java.lang.NullPointerException
	at dao.Sql2oUserDaoTest.shutDown(Sql2oUserDaoTest.java:41)

